maro/cli/inspector/params.py

```diff
@@ -0,0 +1,28 @@
+from enum import Enum
+
+
+class GlobalFilePaths:
+    ports_sum = "ports_summary.csv"
+    vessels_sum = "vessels_summary.csv"
+    stations_sum = "stations_summary.csv"
+    name_convert = "name_conversion.csv"
+
+
+class GlobalScenarios(Enum):
+    CIM = 1
+    CITI_BIKE = 2
+
+
+class CIMItemOption:
+    basic_info = ["name", "frame_index"]
```

Comment:
the metric groups are strange in both naming and contexts and un-unified order.

why citi bike don't have ``basic_info`` and ``acc_info``?
why there are ``remaining space`` in cim but no ``empty dock`` in citi bike?
why there are "extra cost" and ``failed_return`` in citi bike but no ``opearation cost`` in cim?